src/imagetilesource.js

```diff
@@ -195,6 +195,18 @@
             }
             return context;
         },
+        /**
+         * Free up canvas memory
+         * (iOS 12 or higher on 2GB RAM device has only 224MB canvas memory,
+         * and Safari keeps canvas until its height and width will be set to 0).
+         * @function
+         */
+        freeupCanvasMemory: function () {
```

Comment:
How about we call this `destroy`, so it's more general? We already have that concept and some other objects in the system.